Insights/EventProcessor.swift

```diff
@@ -0,0 +1,15 @@
+//
```

Comment:
Any reason why this protocol isn't in the protocol folder? also this comment applies for a few other files that are not under a folder/group